src/index.js

```diff
@@ -142,5 +145,7 @@ const parseArray = (value) => {
   return noBrackets.split(/,\s*/)
 }
 
+const parseNumber = (value) => Number(value)
```

Comment:
I'd probably say we should use `parseInt` here:
```suggestion
const parseNumber = (value) => parseInt(value, 10)
```

Reply:
That makes sense! Will change it.